guides/v2.1/javascript-dev-guide/widgets/widget_accordion.md

```diff
@@ -179,6 +179,44 @@ $( "#element" ).accordion( "deactivate", 1 );
 $( "#element" ).accordion( "deactivate", [0,1]);
 ```
 
+## Events {#accordion_events}
+
+The accordion widget is subscribed to the same events as the [Tabs widget]({{page.baseurl}}/javascript-dev-guide/widgets/widget_tabs.html):
+
+-   [beforeOpen callback](#accordion_beforeOpen)
+-   [dimensionsChanged](#accordion_dimensionsChanged)
+
+### `beforeOpen callback` {#accordion_beforeOpen}
+
+Called before the content is opened. 
+
+Example of adding a callback to `beforeOpen` events:
+
+```javascript
+$("#element").on("beforeOpen", function () {
+    // do something before opening the content
+});
+```
+
+### `dimensionsChanged` {#accordion_dimensionsChanged}
```

Comment:
It's not necessary to add header ID overrides. Jekyll automatically generates header IDs at build time.